### ISSUE TITLE
Error with French language

### ISSUE THREAD

**Issue by pingax** [NagiosEnterprises/ncpa#381]
Hi, 

I installed NCPA 2.0.5 on a French Windows 7 x86_64 workstation. I got an error due to some strings which are badly encoded.

It's the same issue as #330. on my workstation, when I go to the web gui, I got this at /api url : 

```
 "timezone": [
                "Paris, Madrid",
                "Paris, Madrid (heure d��t�)"
```

or this : 

` "Connexion au r�seau local* 12": {`

**Comment by jomann09** [comment]
I only made NCPA not crash from translation problems. I am pretty sure that Python 2.7 is having issues with the encoding. I think that Python 3 will fix this in NCPA 3. I still have yet to be able to configure my Windows to make this happen, I am not sure why - even when I set my encoding to another language and my location, region, and text.

**Comment by pingax** [comment]
Ok :)

Do you have any release date for the v3 ? I'll try to build it at home (still in test phase of nagios deploiement).

**Comment by jomann09** [comment]
I do not have a release date, if you give me a week or so I can get you a copy of the in-development version of 3 for you to test and see if the language issues are resolved! If not, I'll look more into what to do to fix it.

**Comment by pingax** [comment]
Hi, 

Some feedback : using v2.0.5 I'm able to grab some infos from my Windows server. This encoding issue doesn't interfere. Unfortunately, I won't be able to get stats of the network, as it's named "Connexion au réseau local", which npca considers as non-existant.

**Comment by jomann09** [comment]
While this will be solved when we release NCPA 3 I think that I have the issues you mentioned fixed for 2.0.6 and I will test it a little more to verify but so far it seems good. I'm closing this for 2.0.6.
